docs/app-directory/overview.md

```diff
@@ -7,18 +7,75 @@ hide_title: true
+
+
+#### Updating apps
+
+Typically, software evolves over time. The app versions you are running today will not be the same ones you need tomorrow. Therefore, you will need to upgrade apps periodically. Very few people look forward to upgrading, but appD and web deployment can make it easier for you. To roll out a new version of your  app, either update the existing entry for it in appD or add a new entry for that version (allowing users to select the version they will use).
+
+
+#### Agent-agnostic
+
+As a part of the FDC3 standard, appD isn't tied to any specific vendor. This is important, as it allows you more flexibility in that you are not tied to any specific container or desktop agent implementation. If at any point you want to switch to a different desktop agent, the process won’t be prohibitively difficult. The existing appD will work without any additional effort from you, as long as your new desktop agent is also FDC3-compliant. This is in contrast with proprietary solutions, where you would have to reconstruct the entire ecosystem from scratch. \
+ \
+AppD reduces fragmentation in the market and allows end-users more flexibility in what applications can be included in their desktop.
+
+
+#### Intent resolution
+
+AppD provides information to the Desktop Agent on which applications support particular intents and the context types they require as input to them. This allows the Desktop Agent to implement an intent resolver that can launch applications and pass the intent and context to them to operate on, supporting workflows between applications that didn't require prior bilateral agreements between the application providers.
+
+
+### For an Application Provider
+
+Until now, we've looked at appD from the perspective of a desktop owner and user. But appD also offers advantages to vendors who develop apps for the financial servicesdesktop.
+
+
+#### Apps work well together out of the box
+
+When your customers add your FDC3-compliant app to their desktop via an appD record accurately describing it, you can be sure that your app will interoperate with other apps that follow the FDC3 standard. You don’t have to do anything special, or arrange a bilateral agreement with anyone else. The benefit of the open standard is that any app that follows it will work well with any other compliant app.
+
+
+#### Easy updates
+
+As a vendor, you prefer for all your customers to run your latest software. However, many customers will postpone upgrades, sometimes for a long time, because upgrading is a pain. An advantage of a vencdor-hosted appD is that the configuration of app can be updated at any time and, if your customers need to choose when to upgrade, multiple versions of it made available, each with their own configuration. By making it easier for customers to update, you can drive better adoption.
```

Comment:
```suggestion
As a vendor, you prefer for all your customers to run your latest software. However, many customers will postpone upgrades, sometimes for a long time, because upgrading can be a pain. An advantage of a vendor-hosted appD is that the configuration of an app can be updated at any time and, if your customers need to choose when to upgrade, multiple versions of it can be made available, each with their own configuration. By making it easier for customers to update, you can drive better adoption.
```